bin/kmd

```diff
@@ -68,20 +68,13 @@ def main(args):
     if args.dry_run:
         return
 
-    print('Installing {} to {}'.format(args.release, args.prefix))
-    install_root = os.path.join(args.prefix, args.release, 'root')
-
-    komodo.shell('{1} {0} .{0} {0}'.format(args.release, args.renamer))
-    komodo.shell('rsync -a .{} {}'.format(args.release, args.prefix), sudo = args.sudo)
-
-    if os.path.exists('{1}/{0}'.format(args.release, args.prefix)):
-        komodo.shell('{2} {0} {0}.delete {1}/{0}'.format(args.release, args.prefix, args.renamer),
-               sudo = args.sudo)
+    tmp_release = str(uuid.uuid4())
```

Comment:
wouldn't it be a bit more informative to use something like `args.release` followed by date and time?

Reply:
Yes - it is now changed to use `arg-$timestamp` - not very readable; but well suited for sorting.